### ISSUE TITLE
Hello Netflix, h265 is widely available since today

### ISSUE THREAD

**Issue by LifeIsStrange** [Netflix/mantis#274]
@calvin681 @sundargates @hmitnflx 

Hi dear Netflix developers, 
this "issue" has nothing to do with this project however it is of critical importance for you to communicate to your company. 
I have a disruptive news for you that is easy to be unaware of since there has been no dedicated public announcement.

Chromium 107 has been released and bring for the first time h265 (GPU) decoding support, on all main OSes. 
https://chromestatus.com/feature/5186511939567616
Why does this matter?
Well delivering the optimal codec to the end user is Netflix #1 job.
So you'd better start using it anytime it is supported.
Despite all the ideological hype, AV1 is in many cases not superior to h265 regarding compression ratios but most importantly h265 hardware acceleration is order of magnitudes more widespread. As such it is for end users a significant experience improvement (less heat, less FPS stutter/tearing, MUCH more battery)
Second, it is has a gigantic ecological impact, as for you server side it is faster and less resource hungry to compress in h265 but most importantly the significantly reduced electricity [cost]("As you can see a GPU enabled VLC is 70% more energy efficient than using the CPU!"  https://devblogs.microsoft.com/sustainable-software/vlc-energy-optimization-with-gpu/#:~:text=As%20you%20can%20see%20a%20GPU%20enabled%20VLC%20is%2070%25%20more%20energy%20efficient%20than%20using%20the%20CPU!) on users hardware will cut the share of internet on electricity consumption worldwide.
Next step is integrating the already released h266!

TL;DR
Crazy high ecological impact and end user impact (battery, smoothness and size)
I hope you can make Netflix realize this and for the company to start using it in the short term.
